packages/flutter/test/services/platform_views_test.dart

```diff
@@ -276,4 +276,82 @@ void main() {
       );
     });
   });
+
+  group('Linux', () {
+    late FakeLinuxPlatformViewsController viewsController;
+    setUp(() {
+      viewsController = FakeLinuxPlatformViewsController();
+    });
+
+    test('create Linux view of unregistered type', () async {
+      expect(
+        () {
+          return PlatformViewsService.initUiKitView(
+            id: 0,
+            viewType: 'web',
+            layoutDirection: TextDirection.ltr,
+          );
+        },
+        throwsA(isA<PlatformException>()),
+      );
+    });
+
+    test('create Linux views', () async {
+      viewsController.registerViewType('webview');
+      await PlatformViewsService.initLinuxView(
+          id: 0, viewType: 'webview', layoutDirection: TextDirection.ltr);
```

Comment:
Should be `webview` or something Linux related?

Reply:
Android and iOS also take the name of `webview` for testing, to make sure the registration works. This doesn't interact with engine, so any view type is accepted. 
And... I think `webview` is the most common case for platform view.